docs/resources/deployment.md

```diff
@@ -63,12 +63,13 @@ resource "vercel_deployment" "example" {
 
 ### Required
 
-- **files** (Map of String) A map of files to be uploaded for the deployment. This should be provided by a `vercel_project_directory` or `vercel_file` data source.
 - **project_id** (String) The project ID to add the deployment to.
 
 ### Optional
 
 - **environment** (Map of String) A map of environment variable names to values. These are specific to a Deployment, and can also be configured on the `vercel_project` resource.
+- **files** (Map of String) A map of files to be uploaded for the deployment. This should be provided by a `vercel_project_directory` or `vercel_file` data source.
+- **git_source** (Attributes) A map with the Git repo information (see [below for nested schema](#nestedatt--git_source))
```

Comment:
I think we should probably validate that one of `files` or `git_source` is present.

Reply:
on it

Reply:
this has been done in https://github.com/vercel/terraform-provider-vercel/pull/21/files#diff-03202b30898c33935f6776c185f1dd43d67d1d5c6e81fd443eb3a88bb45f4154R212-R220